src/config.js

```diff
@@ -1,3 +1,13 @@
+#-------------------------------------------------------------------------------
```

Comment:
"Default Copyright and License Notice: The standard copyright and license notice should appear in all source files where possible. Variations on this notice may be required (for example where a license other than the CPL or EPL governs the use of the file)." extracted from https://eclipse.org/legal/
Thanks for asking